docker-compose.development.yml

```
@@ -0,0 +1,81 @@
+version: '3.7'
+
+services:
+  flask-server:
+    container_name: flask-server
+    image: newdle
+    build:
+      context: .
+      dockerfile: Dockerfile
+    networks:
+      - default
+    volumes:
+      - './:/opt/newdle'
+      - './docker/development/newdle.cfg.docker:/opt/newdle/newdle/newdle.cfg'
+    ports:
+      - 5000:5000
+    depends_on:
+      postgres:
+        condition: service_healthy
+    environment:
+        - FLASK_RUN_HOST=0.0.0.0
+        - FLASK_ENV=development
+        - FLASK_APP=newdle.core.app
+        - NEWDLE_CONFIG=newdle.cfg
+    working_dir: /opt/newdle
+    entrypoint:
+        - bash
+        - -c
+        - |
+          pip install -q -e '.[dev]'
+          flask db upgrade
+          flask run --extra-files newdle/newdle.cfg
+    user: root
+
+  react-server:
+    container_name: react-server
+    image: newdle
+    networks:
+      - default
+    volumes:
+      - './docker/development/newdle.cfg.docker:/newdle/etc/newdle.cfg'
+      - './newdle/client:/newdle/client'
+    ports:
+      - 3000:3000
+    depends_on:
+      postgres:
+        condition: service_healthy
+      flask-server:
+        condition: service_started
+    working_dir: /newdle/client
+    environment:
+      - FLASK_URL=http://flask-server:5000
+    entrypoint:
+        - bash
+        - -c
+        - |
+            curl -sL https://deb.nodesource.com/setup_12.x | bash -
```

I think we can upgrade this to node 14, as the node-sass version we're using doesn't seem to support it (4.14+):
```suggestion
            curl -sL https://deb.nodesource.com/setup_14.x | bash -
```

---

Are you sure? your dockerfile is using 12, it may lead to inconsistencies